### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by pellared**
#### Bugfixes

- Remove `SPLUNK_METRICS_ENDPOINT` from Instrumentation Libraries
  (it was never really stable).

#### Enhancements

- Add `SPLUNK_PROFILER_MEMORY_ENABLED`.
- Deprecate `jaeger-thrift-splunk` option for `OTEL_TRACES_EXPORTER`.
- Remove the policy regarding Zipkin exporter.
- OTLP exporter can use either `grpc` or `http/protobuf`
  as the default transport protocol.

#### Enhancements

- Add tag protection rule requirement.
- Add Dependabot security configuration requirements.
- Grant Admin role for maintainers team.
- Allow using the documentation public repository
  and reference it in the `CONTRIBUTING.md` template.
- Require documenting all configuration parameters
  that are relevant to Splunk Observability Cloud.
- Require documenting all configuration parameters
  whose default or accepted values deviate from upstream.

#### Breaking Changes

- Remove `telemetry.sdk.language` attribute from `ResourceLogs.resource`.

### Bugfixes

- Remove redunant and conflicting statement about file and line for `ResourceLogs`.

#### Enhancements

- Recommened adding `container.id`, `host.id`, `process.pid` attributes
  to `ResourceLogs.resource`.
- Recommend setting process resource attributes.
- Recommend collecting runtime environment metrics.
- Add a required `profiling.data.total.frame.count` attribute to `LogRecord` for `pprof-gzip-base64`.
- Relax the meaning of `allocation` in `LogRecord` for `pprof-gzip-base64`.

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ga_request.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ assignees: ''
 
 Name and link to repository
 
-### Does the repository follow the latest tagged minor release in [GDI specification](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md)?
+### Does the repository follow the latest tagged minor release in [GDI specification](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md)?
 
-- [ ] Has an appropriate [maintainers team](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md#teams).
-- [ ] [Permissions](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md#permissions)
+- [ ] Has an appropriate [maintainers team](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md#teams).
+- [ ] [Permissions](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md#permissions)
```
Comment (Kielek) [inline .github/ISSUE_TEMPLATE/ga_request.md:17]: This kind of links will no go through lychee validation.
Consider to make it as follow up pr after official release.

**Review by mateuszrzeszutek — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,52 @@
...
+#### Bugfixes
+
+- Remove `SPLUNK_METRICS_ENDPOINT` from Instrumentation Libraries
+  (it was never really stable).
+
+#### Enhancements
+
+- Add `SPLUNK_PROFILER_MEMORY_ENABLED`.
+- Deprecate `jaeger-thrift-splunk` option for `OTEL_TRACES_EXPORTER`.
+- Remove the policy regarding Zipkin exporter.
```
Comment (MrAlias) [inline CHANGELOG.md:18]: I'm not sure these are "enhancements". Maybe they would better fit under "breaking changes"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,52 @@
...
+  as the default transport protocol.
+
+### Repository
+
+#### Enhancements
+
+- Add tag protection rule requirement.
+- Add Dependabot security configuration requirements.
+- Grant Admin role for maintainers team.
+- Refine documentation requirements.
```
Comment (MrAlias) [inline CHANGELOG.md:29]: This is pretty vague. Can we be more specific about what GDI repositories need to change or be aware of here?

File: CHANGELOG.md (hunk 2 above)
Comment (pellared) [reply CHANGELOG.md:18]: I am also not sure if it is a breaking change. The policy was that we should not document Zipkin exporter support.

From 1.4.0 tag:

https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/configuration.md?plain=1#L221-L222

Normally I would put it to "Removed"

Comment (MrAlias) [reply CHANGELOG.md:18]: Agreed, I feel the allowed sections are lacking here. If you don't think there is a better place I'm fine resolving this comment.

File: CHANGELOG.md (hunk 3 above)
Comment (pellared) [reply CHANGELOG.md]: PTAL https://github.com/signalfx/gdi-specification/pull/226/commits/387812410a06f98c9561a1b1a554808b4acf8d14

File: .github/ISSUE_TEMPLATE/ga_request.md (hunk 1 above)
Comment (pellared) [reply .github/ISSUE_TEMPLATE/ga_request.md:17]: https://github.com/signalfx/gdi-specification/pull/226/commits/9dbffff90dae7ad8e325a20cdbde58fc56ed21d8 Done here as I do not want to use "admin powers"

**Review by MrAlias — APPROVED** [review]

**Pull request merged**